runtime/src/cost_model.rs

```diff
@@ -4,21 +4,41 @@
 //!
 //! The main function is `calculate_cost` which returns &TransactionCost.
 //!
-use crate::{block_cost_limits::*, execute_cost_table::ExecuteCostTable};
+use crate::{
+    bank::is_simple_vote_transaction, block_cost_limits::*, execute_cost_table::ExecuteCostTable,
+};
 use log::*;
 use solana_sdk::{pubkey::Pubkey, transaction::SanitizedTransaction};
 use std::collections::HashMap;
 
 const MAX_WRITABLE_ACCOUNTS: usize = 256;
 
 // costs are stored in number of 'compute unit's
-#[derive(AbiExample, Default, Debug)]
+#[derive(AbiExample, Debug)]
```

Comment:
nit: I don't think `AbiExample` is required here, right? We should be able to adjust the `TransactionCost` without worry about compatibility afaict

Reply:
done in #21406